backup/setup/roles.json

```diff
@@ -271,7 +271,8 @@
 			"TASK_DASHBOARD_VIEW_V3",
 			"JOIN_MEETING",
 			"TEAM_CREATE",
-			"TEAM_EDIT"
+			"TEAM_EDIT",
+			"TOOL_CREATE"
```

Comment:
this is not a good point. 
consequence is possibiltiy for student to add tools such as BBB, when this is not wanted